backend/utils_segment_whole_slide.py

```diff
@@ -175,7 +184,7 @@ def process_one_tile(
 
 def create_mosaic(
     tile_grid: List[List[np.ndarray]],
-    tile_size: int
+    tile_size:  TILE_SIZE
```

Comment:
Incorrect: TILE_SIZE is not a type!
```suggestion
    tile_size:  int = TILE_SIZE
```

Reply:
Changed back to int.